parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -72,7 +73,7 @@ async def _load_manifest(self, entry_id: EntryID) -> LocalManifest:
             return manifest
 
     @asynccontextmanager
-    async def _lock_manifest_from_path(self, path: FsPath) -> LocalManifest:
+    async def _lock_manifest_from_path(self, path: FsPath) -> AsyncGenerator[LocalManifest, None]:
```

Comment:
`AsyncIterator[Tuple[LocalManifest, Optional[LocalManifest]]]` to avoid the `None` send type ? (cf. https://docs.python.org/3/library/typing.html#typing.AsyncGenerator)